apache-airflow-mixin/dashboards/airflow-overview.libsonnet

```diff
@@ -0,0 +1,1156 @@
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      decimals: 0,
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'none',
+    },
+  },
+};
+
+local schedulerLogsPanel = {
+  datasource: lokiDatasource,
+  targets: [
+    {
+      datasource: lokiDatasource,
+      editorMode: 'code',
+      expr: '{job=~"$job", instance=~"$instance", dag_file=~"$dag_file"} |= ``',
+      queryType: 'range',
+      refId: 'A',
+    },
+  ],
+  type: 'logs',
+  title: 'Scheduler logs',
+  description: 'Shows the scheduler logs by DAG file.',
+  options: {
+    dedupStrategy: 'none',
+    enableLogDetails: true,
+    prettifyLogMessage: false,
+    showCommonLabels: false,
+    showLabels: false,
+    showTime: false,
+    sortOrder: 'Descending',
+    wrapLogMessage: false,
+  },
+};
+
+
+{
+  grafanaDashboards+:: {
+    'apache-airflow-overview.json':
+      dashboard.new(
+        'Apache Airflow overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+
+      .addTemplates(
+        std.flattenArrays([
+          [
+            template.datasource(
+              promDatasourceName,
+              'prometheus',
+              null,
+              label='Prometheus data source',
+              refresh='load'
+            ),
+          ],
+          if $._config.enableLokiLogs then [
+            template.datasource(
+              lokiDatasourceName,
+              'loki',
+              null,
+              label='Loki data source',
+              refresh='load'
+            ),
+          ] else [],
+          [
+            template.new(
+              'job',
+              promDatasource,
+              'label_values(airflow_scheduler_tasks_executable,job)',
+              label='Job',
+              refresh=1,
+              includeAll=true,
+              multi=true,
+              allValues='',
```

Comment:
```suggestion
              allValues='.+',
```

Reply:
@JonathanWamsley allValues='' seems to work just fine for selectors when All is selected. any specific reason for the suggested change that i'm not quite understanding off the bat?

Reply:
![Screenshot 2023-05-15 at 1 39 24 PM](https://github.com/grafana/jsonnet-libs/assets/13648435/2d35e583-61bd-4e56-8d9c-d033bf1f95b8)

Its a lint suggestion that we have done previously on a number of dashboards, but not all of them. 
![Screenshot 2023-05-15 at 1 43 47 PM](https://github.com/grafana/jsonnet-libs/assets/13648435/5cd67945-e6c9-4a8e-9136-3e83898e0013)